lib/dialects/postgres/data-types.js

```diff
@@ -647,6 +647,39 @@ module.exports = BaseTypes => {
     array_oids: []
   };
 
+  function CIDR() {
```

Comment:
Even these function definitions are not required, as there is no special handling required for Postgres initializing of these datatypes.

Just define OIDs for these types at the top with others

Reply:
Done.